Update WebSocket security BBEs (#3811)
ballerina-platform/ballerina-distribution: examples/websocket-service-basic-auth-file-user-store/websocket_service_basic_auth_file_user_store.md

# WebSocket service - Basic authentication file user store

> The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header in the initial HTTP request against the provided configurations in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. An `websocket:Service` can configure the scopes it needs for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.

Reviewer: ```suggestion
The `websocket:Service` can be secured with basic authentication and optionally, by enforcing authorization. This can be achieved by validating the basic authentication token sent in the `Authorization` header in the initial HTTP request against the provided configurations in the `Config.toml` file. This file stores the usernames and passwords for the authentication and the scopes for the authorization. Configure the scopes required by the `websocket:Service` for authorization in the `auth` field of the `@websocket:ServiceConfig` annotation.
```

> A `401 Unauthorized` response is sent to the client when the authentication fails, and a `403 Forbidden` response is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on user stores.

Reviewer: ```suggestion
A `401 Unauthorized` response is sent to the client when the authentication fails and a `403 Forbidden` response is sent to the client when the authorization fails. Use this to authenticate and authorize requests based on user stores.
```
